### PR TITLE
Multiplayer code test.

### PR DESCRIPTION
**Pull request by IshanAgg125**
Testing the code if multiple people can join the same room.

### CODE REVIEW

File: devsdragons/package.json
Comment (Jericho-R-Orienza) [inline devsdragons/package.json:1]: do lines 25 and 26 not need commas?

**Review by Jericho-R-Orienza — CHANGES_REQUESTED** [review]
in package.json, do lines 25 and 26 need commas? Just double checking

File: devsdragons/package.json
Comment (IshanAgg125) [reply devsdragons/package.json:1]: just line 25

**Comment by AnimishTemple** [comment]
Resolved merge conflicts, and changes suggested by Jericho were fixed on local version of this branch already (these will be resolved in its latest push - along with two people can talk with each other feature.

**Review by Jericho-R-Orienza — APPROVED** [review]

**Review by AnimishTemple — APPROVED** [review]
Nice!

**Pull request merged**